rpcs3/Emu/RSX/GL/GLGSRender.cpp

```diff
@@ -558,7 +455,6 @@ void GLGSRender::on_init_thread()
 	LOG_NOTICE(RSX, "%s", (const char*)glGetString(GL_VENDOR));
 
 	glEnable(GL_VERTEX_PROGRAM_POINT_SIZE);
-	glGetIntegerv(GL_UNIFORM_BUFFER_OFFSET_ALIGNMENT, &m_uniform_buffer_offset_align);
```

Comment:
This is important (at least for NV where 256 alignment is mandated). It only works without it because 512 is a multiple of 256. When allocating uniform constant buffers, use the m_uniform_buffer_offset_align so that we conform to the opengl specification. This line for example should be aligned to uniform_buffer_offset_align https://github.com/RPCS3/rpcs3/pull/2098/files#diff-472ec451195cefea00b28da26e2d4511R633.